103-bundle-runtime.md

```diff
@@ -234,10 +235,11 @@ The `/cnab/app/image-map.json` file mounted in the invocation image will be:
 {
     "my-microservice": {
         "description": "my microservice",
-        "digest": "sha256:aaaaaaaaaaaa...",
-        "image": "technosophos/microservice:1.2.3"
+        "image": "technosophos/microservice@sha256:cca460afa270d4c527981ef9ca4989346c56cf9b20217dcea37df1ece8120687",
+        "originalImage": "gabrtv/microservice@sha256:cca460afa270d4c527981ef9ca4989346c56cf9b20217dcea37df1ece8120687",
+        "digest": "sha256:cca460afa270d4c527981ef9ca4989346c56cf9b20217dcea37df1ece8120687"
```

Comment:
I'm a little thrown off by the digest field. Does it relate to `originalImage` or `image` here?

Reply:
Well spotted! I was assuming that the digest is preserved, as I regard that as a pretty important property of image relocation. If we really want to accommodate non digest preserving relocation, perhaps we need an `originalDigest` field as well.

Reply:
that sounds like a good solution to me!

Reply:
Added.

Reply:
it does not makes sense IMO. The digest should not change depending on the image location